app/src/main/java/org/kiwix/kiwixmobile/zim_manager/FileWritingFileSystemChecker.kt

```diff
@@ -32,9 +32,10 @@ class FileWritingFileSystemChecker : FileSystemChecker {
       when (val capability = readCapability(resultFile)) {
         CAN_WRITE_4GB,
         CANNOT_WRITE_4GB -> return capability
+        else -> {}
```

Comment:
Is adding the remaining branches a good option here 🤔 ?

Reply:
In latest Kotlin we have to provide else otherwise its prompt error.